laurat007/assignment3/WordTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.TreeSet;
+import junit.framework.TestCase;
+import org.junit.Test;
+
+/**
+ *
+ * @author Laura
+ */
+public class WordTest extends TestCase{
+
+    @Test
+    public void testWords1() {
```

Comment:
try naming your tests in more descriptive way: for example "testEmptyGrid" or "testSmallGrid" or whatever. Just test1, test2 doesn't give any context to what is different between them

Reply:
done.